### PR TITLE
new(modern_bpf): add support for select syscall

### PR DESCRIPTION
**Pull request by loresuso**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area API-version

> /area build

> /area CI

> /area driver-kmod

> /area driver-bpf

/area driver-modern-bpf

> /area libscap-engine-bpf

> /area libscap-engine-gvisor

> /area libscap-engine-kmod

> /area libscap-engine-modern-bpf

> /area libscap-engine-nodriver

> /area libscap-engine-noop

> /area libscap-engine-source-plugin

> /area libscap-engine-savefile

> /area libscap-engine-udig

> /area libscap

> /area libpman

> /area libsinsp

> /area tests

> /area proposals

**Does this PR require a change in the driver versions?**

> /version driver-API-version-major

> /version driver-API-version-minor

> /version driver-API-version-patch

> /version driver-SCHEMA-version-major

> /version driver-SCHEMA-version-minor

> /version driver-SCHEMA-version-patch

**What this PR does / why we need it**:
Add support for `select` syscall in modern bpf probe. 

**Which issue(s) this PR fixes**:

Part of #723 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: driver/modern_bpf/definitions/events_dimensions.h [hunk 1]
```diff
@@ -164,6 +164,8 @@
 #define SEMGET_X_SIZE HEADER_LEN + sizeof(int64_t) + PARAM_LEN 
 #define SEMCTL_E_SIZE HEADER_LEN + sizeof(int32_t) * 3 + sizeof(uint16_t) + PARAM_LEN * 4
 #define SEMCTL_X_SIZE HEADER_LEN + sizeof(int64_t) + PARAM_LEN
+#define SELECT_E_SIZE HEADER_LEN
```
Comment (Andreagit97) [inline driver/modern_bpf/definitions/events_dimensions.h:167]: gg :trollface:

**Review by Andreagit97 — APPROVED** [review]
/approve

File: driver/modern_bpf/definitions/events_dimensions.h (hunk 1 above)
Comment (Andreagit97) [reply driver/modern_bpf/definitions/events_dimensions.h:167]: new strategies to avoid conflicts :joy:

Comment (loresuso) [reply driver/modern_bpf/definitions/events_dimensions.h:167]: 🤣

**Review by FedeDP — APPROVED** [review]
/approve

**Review by hbrueckner — APPROVED** [review]

**Pull request merged**